docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -191,7 +192,7 @@ public class CustomTenantResolver implements TenantConfigResolver {
             config.setApplicationType(ApplicationType.HYBRID);
             return Uni.createFrom().item(config);
         } else {
-            // resolve to default tenant config
+            context.put(OidcUtils.TENANT_ID_ATTRIBUTE, OidcUtils.DEFAULT_TENANT_ID);
```

Comment:
@jedla97 Custom resolvers are not required to set the default tenant id manually, returning null is enough

Reply:
@jedla97 I'd not worry about it, I think it is there to support a test where the user switche between multiple realms. It is not something that we should recommend, manually deal with setting the default property.
This specific case is covered at the very end of https://quarkus.io/guides/security-openid-connect-multitenancy#tenant-resolution-for-web-app

Reply:
@sberyozkin Ok I'll leave it as it is now.